src/main.rs

```diff
@@ -313,6 +313,11 @@ fn find_ram_in_linker_script(linker_script: &str) -> Option<MemoryEntry> {
     for (index, mut line) in linker_script.lines().enumerate() {
         line = line.trim();
         line = eat!(line, "RAM");
+
+        while !line.starts_with(":") {
```

Comment:
could you add a comment here that the goal of the code is to skip the region "attributes" (the `(rwx)` part)?